src/Generators/Microsoft.Gen.AutoClient/Emitter.cs

```diff
@@ -440,6 +462,12 @@ private void EmitStatics(RestApiType restApiType, string staticsName)
         OutLn($"private static class {staticsName}");
         OutOpenBrace();
 
+        OutLn("public static readonly char[] Slashes = { '/', '\\\\' };");
+        OutLn();
+
+        OutLn("public const string ValidationExceptionMessage = \"The value can't contain '\\\\', '/', be empty, null or contain only dots (.).\";");
```

Comment:
I'm curious if it's a common pattern across dotnet to make the exception messages as a public constant? Struggling to understand the value...

If it is to allow for testing this exception is thrown, perhaps creating a "AutoClient.RuntimePathValidationException" or something similar would make sense...